tests/kernel/proc_sys_dump.pm

```diff
@@ -0,0 +1,74 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Partially dump /sys and /proc
+# Maintainer: Richard Palethorpe <rpalethorpe@suse.com>
+
+use 5.018;
+use warnings;
+use base 'opensusebasetest';
+use testapi qw(is_serial_terminal :DEFAULT);
+use utils;
+use Time::HiRes qw(clock_gettime CLOCK_MONOTONIC);
+use File::Basename 'basename';
+use JSON;
+use serial_terminal;
+require bmwqemu;
+
+sub run {
+    my ($self)     = @_;
+    my $tar        = '/tmp/tar.xz';
+    my $ps_dump    = 'proc_sys_dump.sh';
+    my $white_list = '~/proc_sys_whitelist.txt';
+
+    assert_script_run("curl -sS -o /tmp/$ps_dump https://raw.githubusercontent.com/richiejp/ltp/dump/scripts/$ps_dump");
```

Comment:
For the time being a script in a personal fork might be ok but that should not be the way to go for later. How about submitting the script to https://github.com/linux-test-project/ltp first?

Reply:
ja, naturlisch. But it might take a long time to get into the LTP, so this is the best way to test it.

Reply:
Also I might rewrite this in C, if it works OK.